lib/default-config.js

```diff
@@ -877,6 +877,10 @@ module.exports = function (config) {
       return field.required ? true : false;
     },
 
+    fieldHasSearch: function (field) {
+      return field.disableSearch !== true ? true : false;
```

Comment:
@neilff Can we reverse the logic like this:

```js
return _.isUndefined(field.enableSearch) ? true : false;
```

or `hasSearch` would be okay too. (Formatic has a mess of conventions with booleans right now, so you'll notice the boolean prefixes are not consistent. 😞 Guess we might as well move toward boolean prefixes though.)

I wish I had done this in the first place, and then I'd force us to just enable search when we want it. I don't want to break everything, so hence keeping the default as `true`.